app/src/main/java/com/sunasterisk/travelapp/data/repository/HotelRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.sunasterisk.travelapp.data.repository
+
+import com.sunasterisk.travelapp.data.OnDataCallback
+import com.sunasterisk.travelapp.data.models.Hotel
+
+interface HotelRepository {
+    fun searchHotelsByProperty(
+        prameters: Map<String, String>,
```

Comment:
những cái param này em viết ra cho nó clear 
```suggestion
        localId: String,
        restaurantId: String,
        ....
```

Reply:
Vì tầng logic bên trên không cần biết các đường path của api